web/annotation_layer_builder.css

```diff
@@ -93,6 +93,60 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation div,
+.annotationLayer .radioButtonWidgetAnnotation div {
+  background-color: rgba(0, 54, 255, 0.13);
```

Comment:
Is this styling also how it looks in Adobe Reader? If not, I'd prefer we keep it the same as Adobe Reader for now as that's what most users will expect.